dipy/viz/horizon/util.py

```diff
@@ -151,7 +151,7 @@ def _unpack_data(data, return_size=3):
     return result
 
 
-def is_binary_image(data, unique_points=100):
+def is_binary_image(data, sample_cube_size=15):
```

Comment:
`sample_cube_size` instead of a fix value, I would recommend to take a percentage of the volume.

`sample_cube_size=15` is not the same if the volume is 100x100x100 or 1000x1000x1000

Reply:
Make sense, I have updated that!